docs/styles.scss

```diff
@@ -105,6 +105,12 @@ pre {
         }
     }
 
+    li.is-disabled{
+        color: rgba(128, 128, 128, 0.4);
+        cursor: not-allowed;
+        text-decoration: none;
+    }
+
```

Comment:
It is actually wrong library, it resolves issue only inside docs. Please put there changes to [fundamental-styles](https://github.com/SAP/fundamental-styles) library.

Reply:
I agree, I am making the changes in https://github.com/SAP/fundamental-styles